lib/facter/resolvers/os_release.rb

```diff
@@ -85,6 +85,8 @@ def join_os_name
           @fact_list[:name] = if os_name.downcase.start_with?('red', 'oracle', 'arch', 'manjaro')
                                 os_name = os_name.split(' ')[0..1].join
                                 os_name
+                              elsif os_name.downcase.end_with?('mariner')
+                                os_name = os_name.split(' ')[-1].strip
```

Comment:
To make rubocop happy, I think you will need to explicitly return `os_name` as is done in the previous condition, or not assign it at all, in the subsequent condtion.

Reply:
You are correct, confirmed removing the assignment causes it to pass rubocop.

Reply:
Pushed fix, it passes rubocop and rspec.